.github/workflows/macos.yml

```diff
@@ -22,28 +22,27 @@ jobs:
       NUM_PROCESSORS: 12
     steps:
       - name: checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
Can this update be made in the other workflows as well? v3 is deprecated.

Reply:
Yes, but because they are low priority, I will get to them in another PR so that this can be merged now.

Reply:
https://github.com/XRPLF/rippled/pull/4949